edx_rest_framework_extensions/permissions.py

```diff
@@ -144,6 +144,26 @@ def _get_user_filter(self, request):
         return None
 
 
+class LoginRequired(IsAuthenticated):
```

Comment:
Consider renaming this to `LoginRedirectIfUnauthenticated` class.

Reply:
Done.